src/evo/deterministicmns.cpp

```diff
@@ -1251,6 +1251,7 @@ static bool CheckService(const ProTx& proTx, TxValidationState& state)
         return true;
     // Shouldn't be possible during self-checks
     case NetInfoStatus::BadInput:
+    case NetInfoStatus::MaxLimit:
```

Comment:
I just noticed ASSERT_IF_DEBUG here. Does it mean if I use debug version and someone will send me a broken p2p message with invalid protx transaction, I will get here assert here?

Reply:
They're commented as not being possible as they are return codes that aren't emitted by `ValidateService()` (and therefore, `Validate()`) at all ([source](https://github.com/dashpay/dash/blob/c3e48ab843ce42c6c6aa51e7db061036ee3839bf/src/evo/netinfo.cpp#L160-L179)).

They can only be emitted by `AddEntry()` (see below).

https://github.com/dashpay/dash/blob/c3e48ab843ce42c6c6aa51e7db061036ee3839bf/src/evo/netinfo.cpp#L183-L185

https://github.com/dashpay/dash/blob/c3e48ab843ce42c6c6aa51e7db061036ee3839bf/src/evo/netinfo.cpp#L190-L193

Should probably replace it with a proper `assert` as on release builds, it'll fall through to the `assert(false)` below regardless (`ASSERT_IF_DEBUG` evaluates to blank on release builds).